### ISSUE TITLE
look at this 

### ISSUE THREAD

**Issue by ROBIKALIVE13** [luckydog7/Funkin-android#518]
how did i find this? xd
take it then be happy:

https://drive.google.com/file/d/1crNKQVT7wYge-HBts0RVXAqN2l_0nrpN/view?usp=sharing

some pics about this
![Screenshot_2021-04-26-09-43-12-975_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82233686/116047281-36c17f80-a674-11eb-9b0d-f35f8304456c.jpg)

![Screenshot_2021-04-26-09-43-27-530_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82233686/116047303-3aed9d00-a674-11eb-82ef-bd2d396cf17e.jpg)

![Screenshot_2021-04-26-09-43-32-158_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82233686/116047313-3d4ff700-a674-11eb-9073-7f696229d058.jpg)

![Screenshot_2021-04-26-09-43-43-136_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82233686/116047330-417c1480-a674-11eb-9d9a-2a38d46ead7b.jpg)

![Screenshot_2021-04-26-09-43-46-746_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82233686/116047346-44770500-a674-11eb-9b65-fa6e4b8b47d5.jpg)

**Comment by ROBIKALIVE13** [comment]
one more thing:
![Screenshot_2021-04-26-09-47-34-069_com.ninjamuffin99.funkin.jpg](https://user-images.githubusercontent.com/82233686/116047541-7720fd80-a674-11eb-9ff1-90373bd003e3.jpg)

**Comment by MrTks216** [comment]
oh, i already have this mod :-)

**Comment by MrTks216** [comment]
but there are some mod can't not play, it will crash.

**Comment by ROBIKALIVE13** [comment]
> but there are some mod can't not play, it will crash.

yh ik its sad

**Comment by kffwlp** [comment]
Could you make a lite version of this? If not it's ok

**Comment by ROBIKALIVE13** [comment]
> Could you make a lite version of this? If not it's ok

im sorry but im not the mod creator

**Comment by kffwlp** [comment]
> > Could you make a lite version of this? If not it's ok
> 
> im sorry but im not the mod creator

Oh it's okay

**Comment by AnthonyCTM** [comment]
Can you tell me who created this? Is so cool and i want to see how it continues

**Comment by DabKnight** [comment]
How the hell did you make this lmao
What a legend

**Comment by Justine101213k** [comment]
Warning This is Virus 330Mb I got big storage it cant install

**Comment by Justine101213k** [comment]
How the hell!!! WOW COOL Can you teach me Who made this mode

**Comment by DanisLixo** [comment]
> How the hell!!! WOW COOL Can you teach me Who made this mode

no, we can't, by the way it is from a discord server (FNF' MOBILE)

**Comment by Justine101213k** [comment]
> no, we can't, by the way it is from a discord server (FNF' MOBILE)

oh can i join discord? Coz i gonna showcase in yt my cahnnel is Gacha Blox

**Comment by AnthonyCTM** [comment]
> > How the hell!!! WOW COOL Can you teach me Who made this mode
> 
> no, we can't, by the way it is from a discord server (FNF' MOBILE)

But you'll keep posting updates for this mod? or not

**Comment by DabKnight** [comment]
Can i join the discord server?
I wanna see those shit ya'll doin lol
Also some issues with mod:
-neo and bside songs do not load.
-only few mod songs work(whitty to be specific)

**Comment by ROBIKALIVE13** [comment]
> Can i join the discord server?
> I wanna see those shit ya'll doin lol
> Also some issues with mod:
> -neo and bside songs do not load.
> -only few mod songs work(whitty to be specific)

its not working for me but i cant fix it bc its not mine

**Comment by AnthonyCTM** [comment]
But you'll keep posting updates for this or not

**Comment by ROBIKALIVE13** [comment]
> But you'll keep posting updates for this or not

if i find any i will

**Comment by AnthonyCTM** [comment]
> > But you'll keep posting updates for this or not
> 
> if i find any i will
N I C E

**Comment by Zizoplays1** [comment]
Funkin forever

**Comment by Zizoplays1** [comment]
Ok, i tried out to play improbable quest and light it up and they instantly crashed

**Comment by AnthonyCTM** [comment]
> Ok, i tried out to play improbable quest and light it up and they instantly crashed
> 
> Edit : dude who made this mod did not even test neo or bsides, both wont work

Well is not complete yet but still you can play all the og songs and the whitty songs + foolhardy so it is pretty complete, we just need patience and we'll have this completed

**Comment by Zizoplays1** [comment]
#557 has some changes by danislixo
